app/src/main/java/org/p2p/wallet/home/MainFragment.kt

```diff
@@ -146,7 +134,7 @@ class MainFragment : BaseFragment(R.layout.fragment_main), MainTabsSwitcher, Cen
                 ScreenTab.EARN_SCREEN -> StubSolendEarnFragment.create()
                 ScreenTab.HISTORY_SCREEN -> HistoryFragment.create()
                 ScreenTab.SETTINGS_SCREEN -> NewSettingsFragment.create()
-                else -> error("No tab found for $clickedTab")
+                ScreenTab.SWAP_SCREEN -> OrcaSwapFragment.create()
```

Comment:
We are adding SWAP to bottom navigation?! why?

Reply:
Design requirement. 
No conflict with Solend is made